### DISCUSSION TITLE
If I wanted a feed I would use FaceBook. Please remove.

### DISCUSSION THREAD

**Discussion by richardklafter** [community/community#13430] [Other Feature Feedback, Questions, & Ideas]
If I wanted a feed I would use FaceBook. Please remove.

**Comment by KlausEverWalkingDev** [comment]
Just not use it :) Problem solved!

**Reply by richardklafter** [reply]
You have every social network on your profile page. Of course you want GitHub to be another social network. It gives you more reach/audience.

**Reply by KlausEverWalkingDev** [reply]
Stop being a gatekeeper. I insist: just don't use it. Or, as I guess you're a dev, create a browser extension to hide it. Again, problem solved :)

**Reply by adriano66** [reply]
How about disabling is by default and letting people who need gazillion of same looking, "social" platforms to enable it if they need it so much.

**Reply by flatrocks** [reply]
The problem will be "solved" when it is possible to make it NOT the time-wasting front page whenever visiting Github.

**Reply by DanielGoldfarb** [reply]
How can you "not use it" ... it shows up in the middle of the page taking up half the page?!   Yes, to not use it I've had to train myself to ignore half the page.  What a waste of time.   Maybe, *just maybe,* if users could customize what does and does not go into the feed, then maybe it would be useful.

**Comment by ghost** [comment]
I don't get it why this feature exists in the first place. There are many other tools on GitHub to reach more audience e.g. "Explore", "GitHub Topics". Why introduce a new one that only causes distraction?

**Comment by fenilli** [comment]
And what do you think your dashboard could look like? just open up your profile when you enter? what solution do you came up with?

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
